grapher/axis/Axis.ts

```diff
@@ -560,6 +560,7 @@ interface DualAxisProps {
     bounds?: Bounds
     horizontalAxis: HorizontalAxis
     verticalAxis: VerticalAxis
+    horizontalAxisOnTop?: boolean
```

Comment:
We could make the orientation of the axis part of the axis config.

I've done it on the `marimekko-axis-orient` branch: https://github.com/owid/owid-grapher/compare/marimekko..marimekko-axis-orient

Let me know what you think!